src/main/java/seedu/address/commons/core/index/Index.java

```diff
@@ -2,7 +2,7 @@
 
 /**
  * Represents a zero-based or one-based index.
- *
+ * <p>
```

Comment:
Same p tag

Reply:
Weird. I think its a side-effect of the IntelliJ autoformat